.travis.yml

```diff
@@ -17,41 +14,15 @@ script:
   - go generate
   - make
 
-  # Build and test the Docker image. It's pushed to Docker Hub as part of the
-  # deploy phase.
-  - docker build -t "$DOCKER_REPO:${TRAVIS_COMMIT:0:8}" .
-  - docker run -d --name stripe-mock-container -p 12111:12111 "$DOCKER_REPO:${TRAVIS_COMMIT:0:8}"
-  - docker ps -a
```

Comment:
Hm, just a note that there was some value to this — if you don't guarantee the integrity of the image here, then the only other way you find out something went wrong is finding out when a downstream build fails.